src/Network/Network/ChangeLog.md

```diff
@@ -27,6 +27,9 @@
     - `New-AzLoadBalancerBackendAddressPool`
     - `Set-AzLoadBalancerBackendAddressPool`
 * Onboard Azure Resource Manager to Private Link Common Cmdlets
+* Support for Sku, ScaleUnits parameters of BastionHost resource.
```

Comment:
Please move this to `Upcoming Release` section.

Reply:
Done